<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Billify</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="Inventory_side" stretch="0,0,0">
        <item>
         <layout class="QHBoxLayout" name="search_boxes" stretch="1,0,1">
          <item>
           <layout class="QHBoxLayout" name="Name_search" stretch="1,3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Find Name : </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="searchNameLineEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Policy::Maximum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="ID_search" stretch="1,5">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Find ID :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="searchIDLineEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="outputTableWidget">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="RefreshButton">
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Bill_tab">
       <attribute name="title">
        <string>Make Bills</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="Bill_side" stretch="0,6,0,0,0,1">
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>Leelawadee UI</family>
              <pointsize>18</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LayoutDirection::LeftToRight</enum>
            </property>
            <property name="text">
             <string>Bill</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="billOutputTableWidget">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="billTotalLabel">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>18</pointsize>
              <fontweight>ExtraBold</fontweight>
             </font>
            </property>
            <property name="text">
             <string>Total</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="BillsCheckBox">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>12</pointsize>
              <italic>false</italic>
              <strikeout>false</strikeout>
              <stylestrategy>PreferDefault</stylestrategy>
              <hintingpreference>PreferVerticalHinting</hintingpreference>
              <fontweight>DemiBold</fontweight>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::FocusPolicy::StrongFocus</enum>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LayoutDirection::RightToLeft</enum>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Get a PDF For Bill  </string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="cleanBillButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Clean the Bill</string>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="getTheBillButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Get the Bill</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Inventory_tab">
       <attribute name="title">
        <string>Edit Inventory</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,11,1,0,2">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>Leelawadee UI</family>
            <pointsize>18</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Inventory Editor</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="logTextEdit">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="addItemButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>13</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Add Item</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editItemButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>13</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Edit Item</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeItemButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>13</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Remove Item</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="InventoryCheckBox">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <italic>false</italic>
            <bold>true</bold>
            <underline>false</underline>
            <strikeout>false</strikeout>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LayoutDirection::RightToLeft</enum>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Get PDF for Inventory Logs  </string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="tristate">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="undoChangesButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Undo Changes</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveChangesButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Save Changes</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1007</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBillsSave">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionChange_Shop_Name"/>
   </widget>
   <widget class="QMenu" name="menuSave_Logs">
    <property name="title">
     <string>Save</string>
    </property>
    <addaction name="actionSave_Bills_Logs"/>
    <addaction name="actionSave_Inventory_Logs"/>
   </widget>
   <widget class="QMenu" name="menuLocation">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_Location"/>
    <addaction name="actionChange_Save_location"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_Application"/>
   </widget>
   <addaction name="menuLocation"/>
   <addaction name="menuBillsSave"/>
   <addaction name="menuSave_Logs"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionInventory">
   <property name="text">
    <string>Inventory Manager</string>
   </property>
  </action>
  <action name="actionBill">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Bill Manager</string>
   </property>
  </action>
  <action name="actionSave_Location">
   <property name="text">
    <string>Goto Save Location</string>
   </property>
  </action>
  <action name="actionChange_Save_location">
   <property name="text">
    <string>Set Save Location</string>
   </property>
  </action>
  <action name="actionLoad_Data_From">
   <property name="text">
    <string>Load Data From</string>
   </property>
  </action>
  <action name="actionChange_Shop_Name">
   <property name="text">
    <string>Update Shop Name</string>
   </property>
  </action>
  <action name="actionSave_Bills_Logs">
   <property name="text">
    <string>Save Bills Logs</string>
   </property>
  </action>
  <action name="actionSave_Inventory_Logs">
   <property name="text">
    <string>Save Inventory Logs</string>
   </property>
  </action>
  <action name="actionAbout_Application">
   <property name="text">
    <string>About Application</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
